aea/cli/utils/config.py

```diff
@@ -221,3 +221,16 @@ def handle_dotted_path(value: str) -> Tuple:
 
     config_loader = ConfigLoader.from_configuration_type(resource_type_plural[:-1])
     return json_path, path_to_resource_configuration, config_loader
+
+
+def update_item_config(item_type: str, package_path: Path, **kwargs):
+    item_config = load_item_config(item_type, package_path)
+    for key, value in kwargs.items():
+        setattr(item_config, key, value)
+
+    config_filepath = os.path.join(
+        package_path, item_config.default_configuration_filename  # type: ignore
+    )
+    loader = ConfigLoaders.from_package_type(item_type)
+    with open(config_filepath, "w") as f:
+        loader.dump(item_config, f)
```

Comment:
Are we sure we do not already have this kind of method as part of the set_config functionality?

Reply:
Didn't find any. And consulted with @marcofavorito , we were missing this kind of method.